machine_learning/DegradationModel.py

```diff
@@ -351,7 +351,7 @@ def __init__(
 
         """ Primitive Dictionary Layer variables """
         self.dry_cell_direct = PrimitiveDictionaryLayer(
-            num_features=6, id_dict=dry_cell_dict
+            num_feats =6, id_dict=dry_cell_dict
```

Comment:
I prefer num_features

Even better would be to have a universal convention for things like number of x